assets/js/custom/AdminMail.js

```diff
@@ -2,14 +2,15 @@
 
 // eslint-disable-next-line no-unused-vars
 function AdminMail () {
-  $('.btn').click(function () {
+  $('.btn-send').click(function () {
```

Comment:
It would be great to refactor this code without jquery, since it is already touched in this MR - (should be do-able by chatgpt in a few seconds ;)